pkgs/top-level/all-packages.nix

```diff
@@ -8060,8 +8060,9 @@ in
 
   scmpuff = callPackage ../applications/version-management/git-and-tools/scmpuff { };
 
-  scream-receivers = callPackage ../misc/scream-receivers {
+  scream = callPackage ../applications/audio/scream {
     pulseSupport = config.pulseaudio or false;
```

Comment:
```suggestion
```

Reply:
Looking at other packages, `config.pulseaudio or stdenv.isLinux` is more common and sensical, so changed to that.